tests/createTimestamps.test.ts

```diff
@@ -35,8 +42,8 @@ const defaultCues = [
     startTime: null,
     duration: 15 * 60000, // 15 min
   },
-]
-const defaultCueOrder = [{ id: '#1' }, { id: '#2' }, { id: '#3' }]
+] as unknown as RundownCue[]
```

Comment:
```suggestion
] as RundownCue[]
```
Or is the `unknown` crucial?

Reply:
https://github.com/rundown-studio/package-utils/pull/6/commits/cf9238bcffadd22a856f2f8559cd06b05f0b85b4